addons/magazinerepack/stringtable.xml

```diff
@@ -66,7 +67,7 @@
             <German>%1 volle(s) Magazin(e) und %2 übrig gebliebene Patrone(n)</German>
             <Spanish>%1 cargador(es) completo(s) y %2 bala(s) extra(s)</Spanish>
             <French>%1 chargeur(s) plein(s) et %2 cartouche(s) en rab</French>
-            <Polish>Pełnych magazynków: %1.&lt;br /&gt; Dodatkowych naboi: %2.</Polish>
+            <Polish>Pełnych magazynków: %1.&lt;br /&gt;Dodatkowych naboi: %2.</Polish>
```

Comment:
Since I had to slice it to parts to sound logical, part after <br /> is in next line. Removed space because it's not needed and with it text will be moved by one space right?